theorems/T000310.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: T000310
+if:
+  P000003: true
+then:
+  P000101: true
+refs:
+  - mo: 191016
+    name: '"All retracts are closed" as separation axiom'
+---
+
+See {{mo:191016}}.
```

Comment:
https://math.stackexchange.com/questions/805274 seems a better reference as it gives a proof.